plugins/swagger-routes.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const util = require('../lib/nodejs/hexo-util');
+
+const getFilter = (hexo) => {
+
+  const swaggerStore = require('../lib/nodejs/swagger-store')({hexo});
+
+  return () => {
+    const routes = swaggerStore.getRoute();
+    routes && Object
+      .keys(routes)
+      .forEach((route) => {
+        const data = routes[route];
+
+        if(data){
+          hexo.route.set(route, data);
+        }else{
+          hexo.route.remove(route);
+        }
+      })
+  }
+}
+
+
+module.exports = ({hexo}) => {
+  const {themeConfig} = util({hexo});
```

Comment:
@bhaskarmelkani why do you need this?

Reply:
Removed. First I was adding some configs for routes then removed it for simplicity.